Add footer

nastyawaks opened the pull request:
~~Потрібно змінити налаштування фонових зображень, вони не відповідають макету~~ 
~~Навігація в таблет та декстоп не відцентрована по вертикалі~~

nastyawaks committed:
add footer

ukrwebprom commented on src/sass/_footer.scss:
```diff
@@ -1,19 +1,205 @@
 .footer {
```
после того как сонтейнер перенесли внутрь секций, эти стили с фоновыми картинками нужно назначить не для  footer а для footer-container

ukrwebprom commented on src/sass/_footer.scss:
```diff
@@ -1,19 +1,205 @@
+
+.footer-container {
+  @media screen and (max-width: 767px) {
+    width: 320px;
```
это не нужно. ширина контейнера для мобильного устройства 480пикс или 100% ширины экрана если экран уже 480пикс.

nastyawaks replied:
Убрала, но тогда меню тянется по ширине контейнера и форма с кнопкой тоже. Можно ограничить размер поля формы, но смотрится странно

ukrwebprom commented on src/sass/_footer.scss:
```diff
@@ -1,19 +1,205 @@
+  }
+}
+
+.footer-container {
```
контейнеру нужно добавить левый и правый паддинг
16пикс для десктопа, 32 пикс для планшета и 20 пикс для телефона

nastyawaks replied:
там класс “container footer-container”, поэтому паддинги есть

nastyawaks committed:
footer update with bck images

ukrwebprom commented on src/sass/_footer.scss:
```diff
@@ -1,19 +1,237 @@
+      background: url(../images/footer-back-bottom-tablet@2x.png) no-repeat
+          bottom center,
+        url(../images/footer-back-right-tablet@2x.png) no-repeat right top;
+      background-size: 100% auto, 203px 365px;
```
background-size можно не указывать. картинки все нужных размеров и они будут отображаться в оригинальном размере.

nastyawaks replied:
Пробовала не указывать и тогда 2х становятся очень большими

ukrwebprom replied:
да, ты права. но тогда надо убрать 100% auto чтобы не тянулось на таблетке. я сейчас закину новую картинку без обрезанного яблока справа

ukrwebprom replied:
есть. обнови майн. сейчас для планшета есть нижнее изображение (не обрезанное) и отдельно правое

nastyawaks replied:
Я вчера закинула нижнее и правое для таблет и 2х, т.к ранее там была одна, поэтому смогла отцентровать их (теперь есть конфликт).
Разве задать размеры будет ошибкой?

ukrwebprom replied:
нет, но у тебя в планшетной версии было 100% и нижняя картинка тянулась

ukrwebprom replied:
по поводу конфликта, когда будешь отправлять pull request и гит выдаст конфликт, там в выпадающем списке выбери свои варианты

ukrwebprom reviewed (changes requested):
- меню не отцентровано по вертикали относительно логотипа
- у тебя заданы нижние отступы и в .footer-sub и в .footer__wrap. надо оставить только .footer__wrap и прописать для разных размеров экрана.
- в целом, я бы сделал чуть по другому - у тебя сейчас в .footer__wrap 3 дочерних элемента, поэтому их сложно позиционировать на разных размерах. что если сделать еще одну обертку в которой будет логотип и меню, и отдельно, вторым элементом .footer-sub. думаю так будет проще

nastyawaks committed:
add align for nav, new margins

Обновила фоновую картинку

Добавила центрирование по вертикали (марджином)

Перенесла отступы из .footer-sub, теперь только в .footer__wrap
